src/client/coll/mod.rs

```diff
@@ -127,7 +127,7 @@ impl<'a> Collection<'a> {
     /// Finds the distinct values for a specified field across a single collection.
     pub fn distinct(&self, field_name: &str, filter: Option<bson::Document>, options: Option<DistinctOptions>) -> Result<Vec<Bson>> {
 
-        let opts = options.unwrap_or(DistinctOptions::new());
+        let _opts = options.unwrap_or(DistinctOptions::new());
```

Comment:
The only option for `distinct` is maxTimeMs, which doesn't seem to be supported in the current op spec. Looking at the new `find` spec drafts, I assume this is for future 3.2+ support; the underscore hides the warning and allows us to easily add support later (standard rust style).